lib/kafo/scenario_options.rb

```diff
@@ -0,0 +1,66 @@
+module Kafo
+  # A class containing constants for all scenario options
+  class ScenarioOptions
```

Comment:
Before I rebase: should this be singular or plural? I think `Kafo::ScenarioOption::NAME` reads better than `Kafo::ScenarioOptions::Name`. Thoughts?

Reply:
Singular.